docs/src/content/ignition/docs/guides/creating-modules.md

```diff
@@ -307,6 +309,8 @@ const token = m.contract("Token", ["My Token", "TKN2", 18], { from: account1 });
 
 ## Using existing artifacts
 
+> definition of "artifact" - an ABI? or... does this refer to the exact format of a hardhat artifact? link to a definitions? This documentation is excellent, and can be clearly understood. Artifact is a super abstract word in normal english.
```

Comment:
And by artifact we mean a Hardhat compilation artifact: https://hardhat.org/hardhat-runner/docs/advanced/artifacts#compilation-artifacts